File: trail1/배수의 개수 2/number-of-multipliers-2.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a;
    int k=0;

    for(int i=0;i<10;i++){
        cin>>a;
        if(a%2==1){
            k++;
        }
    }
    cout<<k;
    return 0;
}

File: trail1/배수의 개수 2/number-of-multipliers-2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.

        Scanner scin=new Scanner(System.in);

        int k=0;

        for(int i=0;i<10;i++){
            int a=scin.nextInt();
            if(a%2==1) k++;
        }

        System.out.print(k);
    }
}